### PR TITLE
Merge request Approvals API

### LINKED ISSUE #392: Implement the Project MR approvals
Hey guys!

This endpoint is really missing for us: any chance you could include it?

Cheers!

Gitlab endpoint documentation: https://docs.gitlab.com/ee/api/merge_request_approvals.html#change-configuration

### PR DESCRIPTION
**Pull request by akkee**
Closes #392

### CODE REVIEW

File: lib/gitlab/client/merge_request_approvals.rb [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+    # Get Configuration for approvals on a specific Merge Request.
+    #
+    # @example
+    #    Gitlab.merge_request_mr_approvals_configuration(1, 5)
+    #
+    # @param [Integer] project(required) The ID of a project.
+    # @param [Integer] merge_request(required) The IID of a merge_request.
+    # @return [Gitlab::ObjectifiedHash] MR approval configuration information about the merge request
+    def merge_request_mr_approvals_configuration(project, merge_request)
```
Comment (PatrickLerner) [inline lib/gitlab/client/merge_request_approvals.rb]: Do we need the prefix to be `merge_request_mr`? I think `merge_request` would be sufficient.

Comment (akkee) [reply lib/gitlab/client/merge_request_approvals.rb]: So should I edit it or its OK ?

**Comment by PatrickLerner** [comment]
I'm not a maintainer ;) I just came about your MR and am using your code myself. Seems this project is not so actively maintained.

File: lib/gitlab/client/merge_request_approvals.rb (hunk 1 above)
Comment (PatrickLerner) [reply lib/gitlab/client/merge_request_approvals.rb]: I'm not a maintainer ;) I just came about your MR and am using your code myself. Seems this project is not so actively maintained.

Comment (akkee) [reply lib/gitlab/client/merge_request_approvals.rb]: okay @asedge , any chance this gets merged ?

Comment (asedge) [reply lib/gitlab/client/merge_request_approvals.rb]: `merge_request_approvals` would be more consistent with other method naming.

**Comment by born4new** [comment]
@asedge Would it be possible to release a new gem version with that change?

**Pull request merged**